File: src/main/java/superstitio/powers/masoOnly/LostBodyPart_Breast.java
package superstitio.powers.masoOnly;

import superstitio.DataManager;

import static superstitio.cards.general.FuckJob_Card.BodyPart;

public class LostBodyPart_Breast extends LostBodyPart{
    public static final String POWER_ID = DataManager.MakeTextID(LostBodyPart_Breast.class);

    public LostBodyPart_Breast() {
        super(POWER_ID);
    }
    @Override
    public BodyPart[] banedBodyPart() {
        return new BodyPart[]{BodyPart.breast};
    }
}

File: superstitioMain/java/superstitio/powers/masoOnly/LostBodyPart_Breast.kt
package superstitio.powers.masoOnly

import superstitio.DataManager
import superstitio.cards.general.FuckJob_Card.BodyPart

class LostBodyPart_Breast : LostBodyPart(POWER_ID) {
    override fun banedBodyPart(): Array<BodyPart?> {
        return arrayOf(BodyPart.breast)
    }

    companion object {
        val POWER_ID: String = DataManager.MakeTextID(LostBodyPart_Breast::class.java)
    }
}
